src/components/android-settings-container.js

```diff
@@ -45,9 +45,14 @@ class AndroidSettingsContainer extends React.Component {
                 ? (e.target.value = maxValues.valuesTEI)
                 : (e.target.value = valueInput)
         }
+        let value = e.target.value
+        e.target.name === 'encryptDB'
+            ? (value = e.target.checked)
+            : (value = e.target.value)
```

Comment:
These three lines would be cleaner as a simple `if` statement:

```js
if (e.target.name === 'encryptDB') {
    value = e.target.checked
}
```

Reply:
okey dokey, I just corrected those lines and also regenerated `en.pot`